src/components/Task/Difficulty/DifficultyLabel/DifficultyLabel.tsx

```diff
@@ -6,67 +6,52 @@ import { DifficultyEnum } from '../../../../models/difficulty.enum';
 
 interface DifficultyLabelProps {
     difficulty: DifficultyEnum;
+    active?: boolean;
 }
 
 class DifficultyLabel extends React.Component<DifficultyLabelProps> {
     render() {
-        const { difficulty } = this.props;
+        const { difficulty, active } = this.props;
         const colors = colorData.difficultyColor;
         let color;
         if (difficulty in colors) {
             color = colorData.difficultyColor[difficulty];
         } else {
             color = colorData.difficultyColor.UNKNOWN;
```

Comment:
Concider refactor this and don't use colors from the Json. Instead you could use classes with colors.